src/index.html

```diff
@@ -5,8 +5,371 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="imac" href="images/imac.jpeg">
+    <link rel="star" href="./images/star.svg">
+    <link rel="star-active" href="./images/star-active.svg">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          href="#"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link nav__link--active is-active" href="#samsung">apple</a></li>
```

Comment:
check your code style, it should be like that
<img width="941" alt="image" src="https://user-images.githubusercontent.com/102411130/218036557-3d86ddd0-e571-4753-b5b6-60c8d4ef02f6.png">